Jenkinsfile_CNP

```diff
@@ -77,7 +76,11 @@ withPipeline(type, product, component) {
     }
   }
 
-  before('smoketest:saat-staging') {
+  after('prod:promotion') {
+    archiveCoreCaseDataDefinitions('prod')
```

Comment:
Pipeline is changed, there no longer smoke test phase in prod part of the pipeline so this tries to generate XLS straight after build promotion to prod.